### DISCUSSION TITLE
Welcome to Pottery Discussions!

### DISCUSSION THREAD

**Discussion by brainix** [brainix/pottery#290] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hcsama** [comment]
Hi, I am looking for a convenient way to store and retrieve (large, multi level) dicts in Redis. Pottery looks good. What I do not understand how to do is how get the contents for a RedisDict which has previously been stored in Redis. The samples all appear to only show how to create and modify new dicts.
thx
